modules/fingerprint.py

```diff
@@ -114,7 +115,8 @@ def validateFingerPrints(self):
 
         # Validate root partition fingerprints
         if os.path.isfile(root_FingerPrintFile):
-            toReturn = self.do_validateFingerPrints(root_FingerPrintFile, self.root_fingerprints)
+            if not self.do_validateFingerPrints(root_FingerPrintFile, self.root_fingerprints):
+                toReturn = False
         else:
```

Comment:
Can this commit be incorporated in the 4th commit where this change is introduced? Since this is PR level, we can have clean commits merged in the git tree rather than introduce fixes commits like this one